src/main/java/org/example/homework20_glovoapi/service/OrderService.java

```
@@ -0,0 +1,68 @@
+package org.example.homework20_glovoapi.service;
+
+import lombok.AllArgsConstructor;
+import org.example.homework20_glovoapi.model.Item;
+import org.example.homework20_glovoapi.model.Order;
+import org.springframework.stereotype.Service;
+
+import java.util.*;
+
+@AllArgsConstructor
+@Service
+public class OrderService {
+    private final Map<Long, Order> orders = new HashMap<>();
+
+    private final ItemService itemService;
+
+    public Order getOrderById(long id) {
+        if (orders.containsKey(id)) {
+            return orders.get(id);
+        }
+        return null;
```

the result of return orders.get(id); will be the same
also it's better to throw an exception with clear message instead of null